corehq/ex-submodules/casexml/apps/phone/tests/test_ota_restore_v3.py

```diff
@@ -53,3 +54,48 @@ def testUserRestoreWithCase(self):
             dummy_restore_xml(sync_log_id, expected_case_block, items=4),
             restore_payload
         )
+
+
+class TestRestoreResponse(SimpleTestCase):
+    def _expected(self, username, body, items=None):
+        items_text = ' items="{}"'.format(items) if items is not None else ''
+        return (
+            '<OpenRosaResponse xmlns="http://openrosa.org/http/response"{items}>'
+            '<message nature="ota_restore_success">Successfully restored account {username}!</message>'
+            '{body}'
+            '</OpenRosaResponse>'
+        ).format(
+            username=username,
+            body=body,
+            items=items_text
+        )
+
+    def test_no_items(self):
+        user = 'user1'
+        body = '<elem>data0</elem>'
+        expected = self._expected(user, body, items=None)
+        with StringRestoreResponse(user, False) as response:
+            response.append(body)
+            self.assertEqual(expected, str(response), '')
```

Comment:
Its a message but field but I'm pretty sure I didn't add that in there, weird. Will remove.